front50-core/src/main/java/com/netflix/spinnaker/front50/ApplicationPermissionsService.java

```diff
@@ -183,7 +177,7 @@ private void syncUsers(Permission newPermission, Permission oldPermission) {
     if (fiatConfigurationProperties.getRoleSync().isEnabled()) {
       try {
         fiatService.get().sync(new ArrayList<>(roles));
-      } catch (RetrofitError e) {
+      } catch (SpinnakerServerException e) {
```

Comment:
These changes don't make sense until the [FiatService bean](https://github.com/spinnaker/fiat/blob/9c42ba2a2acef3c9e826668e0949ce4721ae7da3/fiat-api/src/main/java/com/netflix/spinnaker/fiat/shared/FiatAuthenticationConfig.java#L80) uses [SpinnakerRetrofitErrorHandler](https://github.com/spinnaker/kork/blob/master/kork-retrofit/src/main/java/com/netflix/spinnaker/kork/retrofit/exceptions/SpinnakerRetrofitErrorHandler.java), with a line of code like
```
          .setErrorHandler(SpinnakerRetrofitErrorHandler.getInstance())
```
but then we need to coordinate this change with all consumers of FiatService.  Sure would love a monorepo for this.  In the meantime, can you identify all the consumers and get PRs ready?  It's gonna be a bit of a trick to get the autobump PRs to not merge automatically while we get all the changes in.

Reply:
And I'm wrong...Apparently since https://github.com/spinnaker/fiat/pull/1034 went in, the FiatService bean does use SpinnakerRetrofitErrorHandler....so this code has been broken / not noticing exceptions since then.

Reply:
I will resolve this since was already covered